package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "solid",
+  "version": "1.0.0",
```

Comment:
We should lower the version. Maybe 0.0.1? Although, semantic versioning does define it as so, most people take 0.1.0 to be the first version you share with others, so IMO wouldn't recommend using 0.1.0 yet

@htharker42 @leekahung @Jared-Krajewski  WDYT?

Reply:
Yeah, I think that's a good suggestion. I don't have a problem with it.

Reply:
Agreed being so early on.

Reply:
updated version